docs/guest.md

```diff
@@ -50,6 +66,51 @@ Please refer to [velero-plugin-for-vsphere tags](https://github.com/vmware-tanzu
 
 If it is an air-gapped environment, please refer to [Install Notes with Customized Images](install-notes.md) for extra required steps.
 
+## Upgrade
```

Comment:
Would you please add an entry for `Upgrade` to  `Table of Contents`?

Reply:
Table of contents was missing here, added it.